### ISSUE TITLE
github-test-reporter as github action

### ISSUE THREAD

**Issue by martinsbalodis** [ctrf-io/github-test-reporter#39]
I think this project would benefit a lot if it was listed in GitHub action marketplace by a verified creator. Right now using latest version from npm feels risky.

**Comment by joshrlesch** [comment]
I agree with this. There isn't a need to install this in our dev dependencies if we can't run it locally. Having it as a Github Action makes a lot of sense.

**Comment by Ma11hewThomas** [comment]
Thanks for suggestion, now available on marketplace
